nipype/interfaces/traits_extension.py

```diff
@@ -218,6 +218,38 @@ def __init__(self, value='', auto_set=False, entries=0,
         super(Directory, self).__init__(value, auto_set, entries, exists,
                                         **metadata)
 
+class ImageFile(File):
+    """ Defines a trait of specific neuroimaging files """
+
+    def __init__(self, value='', filter=None, auto_set=False, entries=0,
+                 exists=False, types=['nii', 'hdr', 'img'], compressed=True,
```

Comment:
the reason i would suggest going with file format types instead of extensions is that programs that accept nifti-1 may not be able to run with nifti-2.

so here is an incomplete list: http://www.reproducibleimaging.org/module-dataprocessing/03-data/ (see mri data section)

and here is a description of nifti-1 vs nifti-2 check for future reference (i don't want to implement all validators right now).